website/layouts/partials/data.html

```diff
@@ -688,6 +688,10 @@ <h4 id="{{ $id }}">
           {{ else }}
           {{ partial "badge.html" (dict "word" "fallible" "color" "yellow" "inline" true "href" $href) }}
           {{ end }}
+          {{/* Deprecation Badge */}}
+          {{ if $func.deprecated }}
+          {{ partial "badge.html" (dict "word" "deprecated" "color" "gray" "inline" true "href" $href) }}
```

Comment:
```suggestion
          {{ partial "badge.html" (dict "word" "deprecated" "color" "red" "inline" true "href" $href) }}
```

I'd suggest making this a brighter color to make it pop out more. I see yellow is already in-use by fallible.